perllib/FixMyStreet/Cobrand/Surrey.pm

```diff
@@ -32,6 +32,17 @@ sub disambiguate_location {
 
 sub report_sent_confirmation_email { 'external_id' }
 
+=item * The default map view shows closed/fixed reports for 31 days
+
```

Comment:
https://github.com/mysociety/societyworks/issues/4391 asks for 90 days for open reports; I am assuming this is a default value but where is it defined in our codebase? Default.pm gives 6 months.

Reply:
Doh, misread that entirely, thanks. Pushed a fixup for that